ranker/modules/omnicorp_overlay.py

```diff
@@ -94,6 +93,7 @@ async def add_shared_pmid_counts(
                     omnisupport = sg
```

Comment:
Can there only ever be one omnicorp support graph per result?

Reply:
If so, though marginal, we should break out of this loop once we find it.

Reply:
Good point